docs/how_to_contribute.rst

```diff
@@ -61,3 +61,24 @@ until all checks pass.
 
 
 Once you got everything to pass, you can stage and commit your changes and push them to the remote github repository.
+
+How to change the documentation
+-------------------------------
+
+In order to build the documentation locally you should build and activate the docs environment::
+
+   conda env create -f docs/environment.yml
+
+   conda activate gcm-filters-docs
+
+Then navigate to the docs folder and build the docs locally with::
+
+   cd docs
+
+   make html
+
+Once that is done you can open the created html files in `docs/_build/index.html` with your webbrowser::
+
+   open _build/index.html
```

Comment:
```suggestion
   cd _build/html; python -m http.server
```

Better to preview the docs through an actual web server. [Sphinx autobuild](https://pypi.org/project/sphinx-autobuild/) is another good option.

Reply:
Thanks! It works for me, though I do get this warning on the "make" command:

`WARNING: html_static_path entry '_static' does not exist`

Not sure what the `python -m http.server` command does. For me it appears to do nothing. But I can just open index.html with my browser instead.

Reply:
I am also just learning about this. @rabernat is there some documentation on this that I could read up on? Happy to change this though.